addons/sale_timesheet/models/project.py

```diff
@@ -249,24 +274,42 @@ def get_panel_data(self):
         panel_data = super(Project, self).get_panel_data()
         return {
             **panel_data,
+            'analytic_account_id': self.analytic_account_id.id,
+            'sold_items': self._get_sold_items(),
             'profitability_items': self._get_profitability_items()
         }
 
+    def get_sale_order_lines(self):
```

Comment:
Why is this method public ?

Reply:
I am calling it as well from enterprise/project_timesheet_forecast_sale. Is there a better way to do it?